.github/workflows/e2e-pr-tests.yml

```
@@ -0,0 +1,290 @@
+
+env:
+  NODE_VERSION: 20.x
+  DO_NOT_TRACK: 1
+
+jobs:
+  test-workflow-trigger:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issue_comment' || github.event_name == 'pull_request_target'
+    steps:
+      - name: Debug trigger
+        run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)